src/flags.rs

```diff
@@ -148,6 +149,11 @@ impl<'this> RustcCodegenFlags<'this> {
                 self.branch_protection =
                     Some(flag_ok_or(value, "-Zbranch-protection must have a value")?);
             }
+            // https://doc.rust-lang.org/beta/unstable-book/compiler-flags/dwarf-version.html
+            // FIXME: Drop the -Z variant and update the doc link once the option is stablized
+            "-Zdwarf-version" | "-Cdwarf-version" => {
+                self.dwarf_version = Some(flag_ok_or(value, "-Zdwarf-version must have a value")?);
```

Comment:
I can imagine us wanting to do different stuff based on the actual version, so maybe we should parse it to an integer here?

Reply:
Maybe add a helper `arg_to_int` or smth, similar to `arg_to_bool`.

Reply:
Done!